app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -228,20 +270,58 @@ class PatcherViewModel(
     fun open() = installedPackageName?.let(pm::launch)
 
     fun install(installType: InstallType) = viewModelScope.launch {
+        var pmInstallStarted = false
         try {
             isInstalling = true
+
+            val currentPackageInfo = pm.getPackageInfo(outputFile)
+                ?: throw Exception("Failed to load application info")
+
+            // If the app is currently installed
+            val existingPackageInfo = pm.getPackageInfo(packageName)
```

Comment:
This needs to account for patches that change the package name.
```suggestion
            val existingPackageInfo = pm.getPackageInfo(currentPackageInfo.packageName)
```